src/elements/common/annotator-context/__tests__/withAnnotations.test.tsx

```diff
@@ -63,12 +77,31 @@ describe('elements/common/annotator-context/withAnnotations', () => {
         });
     });
 
+    describe('componentDidUpdate()', () => {
+        test('should call escalateAnnotationsFetchError and emit event', () => {
```

Comment:
Should we test the other branches of logic to make sure it doesn't emit if no error, or hasn't init, or if it has init but has a new error?